force-app/main/default/lwc/attendanceRow/attendanceRow.html

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        class="slds-box small-box slds-var-m-around_x-small slds-var-p-horizontal_medium"
+    >
+        <lightning-record-edit-form
+            record-id={recordId}
+            object-api-name={serviceDeliveryObject}
+        >
+            <!-- onload={handleLoad}
```

Comment:
Do we need to keep the commented lines.

Reply:
Agree, I don't see a reason to keep commented lines here.